src/server/server_family.cc

```diff
@@ -1577,6 +1577,9 @@ error_code ServerFamily::Drakarys(Transaction* transaction, DbIndex db_ind) {
   transaction->Execute(
       [db_ind](Transaction* t, EngineShard* shard) {
         t->GetDbSlice(shard->shard_id()).FlushDb(db_ind);
+        if (!db_ind || db_ind == DbSlice::kDbAll) {
+          shard->search_indices()->DropAllIndices();
+        }
         return OpStatus::OK;
```

Comment:
1. Can't we call this from FlushDb?
2. Let's try getting rid of this part then https://github.com/dragonflydb/dragonfly/blob/54cf6f0d06dff9b7ffb2dad2c92a2a69e3a0363e/src/server/rdb_load.cc#L2625-L2634